packages/express/src/__tests__/clerkMiddleware.test.ts

```diff
@@ -70,9 +70,6 @@ describe('clerkMiddleware', () => {
 
   it('supports usage without parameters: app.use(clerkMiddleware())', async () => {
     await runMiddleware(clerkMiddleware(), { Cookie: '__clerk_db_jwt=deadbeef;' }).expect(200, 'Hello world!');
-
-    // TODO: Observability headers are not added by default
-    // assertSignedOutDebugHeaders(response);
```

Comment:
This is unnecessary because debug headers is not be injected by default (`enableHandshake` set to `false`)